features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/wso2/templates/overview/template.jag

```diff
@@ -160,7 +160,19 @@
             <% if(api.tags!=''){%>
             <tr>
                 <td><%=i18n.localize("Tags")%></td>
-                <td id="tags"><%=api.tags%></td>
+                <td id="tags">
```

Comment:
please reformat the changes. Seems spaces are not properly added.

Reply:
Fixed in 3a55a4965d6e5aecc169e583ae3f132e6a189b7f